src/transport/PASESession.cpp

```diff
@@ -65,8 +65,11 @@ void PASESession::Clear()
     memset(&mWS[0][0], 0, sizeof(mWS));
     memset(&mKe[0], 0, sizeof(mKe));
     mNextExpectedMsg = Protocols::SecureChannel::MsgType::PASE_Spake2pError;
-    mSpake2p.Init(nullptr);
+
+    // Note: we don't need to explicitly clear the state of mSpake2p object.
```

Comment:
It says 'we don't need to' which sounds a as if we could but it does not matter.

However from the change, it seems that calling 'init' actually caused unit test failues, so we likely we 'must not init spake2p'. Should we update the comment? I think the comment may make sense within this PR, however looking in time the comment could use a bit more context. Like:

```cpp
// clearing mCommisioningHash will automatically clear mSpake2p. 
// NOTE: do NOT call mSpake2p.Init(nullptr) here since that will leak the spake hash.
```

... assuming the above comment is really what is going on.
Could we fix the possiblity of a memory leak when init is called?